UnityProject/Assets/Scripts/HealthV2/Living/Metabolism/MetabolismSystemV2.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Mirror;
+
+public class MetabolismSystemV2 : NetworkBehaviour
+{
+	private int nutritionLevel = 400;
+	public int NutritionLevel => nutritionLevel;
+
+	private HungerState hungerState;
```

Comment:
Codacy found an issue: [Remove unassigned field 'hungerState', or set its value.](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6324510)

Reply:
This will be used when hunger is in